### PR TITLE
Document external configuration parameter values

### PR DESCRIPTION
**Pull request by malteo**

### CODE REVIEW

File: docs/external-configuration-parameter-values.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+`values` field containing the key-value pairs for the configuration parameter
+identifier and value:
+
+```json
+{
+  "values":
+  {
+    "value1": "Value one description",
+    "value2": "Value two description"
+  }
```
Comment (gionn) [inline docs/external-configuration-parameter-values.md]: array for values or ordering would be undefined

File: docs/external-configuration-parameter-values.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Your endpoint will receive a POST request with a JSON body content like this:
+
+```json
+{
+  "productId": 123,
+  "productIdentifier": "your-product",
+  "productVersionId": 456,
+  "productVersionIdentifier": "your-product-plan",
+  "configurationParameter": "CUSTOM_DOMAIN",
+  "language": "en"
```
Comment (gionn) [inline docs/external-configuration-parameter-values.md:23]: cloudCredential? if set

Comment (pimuzzo) [reply docs/external-configuration-parameter-values.md:23]: maybe also cloud provider could be useful

Comment (malteo) [reply docs/external-configuration-parameter-values.md:23]: in another PR, it's missing from the order validation payload too

**Review by manuelmazzuola — APPROVED** [review]

**Review by pimuzzo — APPROVED** [review]

**Pull request merged**